apis/pipelines/v1alpha5/conversion.go

```diff
@@ -1,9 +1,21 @@
 package v1alpha5
 
 import (
+	"github.com/sky-uk/kfp-operator/apis"
 	hub "github.com/sky-uk/kfp-operator/apis/pipelines/v1alpha6"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var (
+	DefaultProvider string
+)
```

Comment:
```suggestion
var DefaultProvider string
```

Reply:
Applied in https://github.com/sky-uk/kfp-operator/pull/394/commits/97c408776a235828d74af923cad516b52bbb4c20